### PR TITLE
feat(uat): add t21 and plugin to generate html-documentation

### PR DESCRIPTION
**Pull request by auarbekov**
**Issue #, if available:**
add and remove steps for loopback address

**Description of changes:**
- Add 'I add IP address "ip_address" to loopback interface' step
- Add 'I remove IP address "ip_address" from loopback interface' step
- Add plugin to generate html-documentation from JavaDoc

**Why is this change necessary:**
Implement scenarios

**How was this change tested:**
Run scenario on CodeBuild

**Test results:**
```
Given my device is registered as a Thing....................................passed
And my device is running Greengrass.........................................passed
When I create a Greengrass deployment with components.......................passed
And I create client device "basic_connect"..................................passed
When I associate "basic_connect" with ggc...................................passed
And I update my Greengrass deployment configuration, setting the component aws.greengrass.clientdevices.Auth configuration to:.passed
And I update my Greengrass deployment configuration, setting the component aws.greengrass.client.Mqtt5JavaSdkClient configuration to:.passed
And I update my Greengrass deployment configuration, setting the component aws.greengrass.clientdevices.IPDetector configuration to:.passed
And I deploy the Greengrass deployment configuration........................passed
Then the Greengrass deployment is COMPLETED on the device after 5 minutes...passed
And the aws.greengrass.clientdevices.mqtt.EMQX log on the device contains the line "is running now!." within 1 minutes.passed
Then I add IP address "127.0.0.2" to loopback interface.....................passed
Then I wait 90 seconds......................................................passed
And I discover core device broker as "default_broker" from "basic_connect" in OTF.passed
And I connect device "basic_connect" on aws.greengrass.client.Mqtt5JavaSdkClient to "default_broker" using mqtt "v5".passed
And I disconnect device "basic_connect" with reason code 0..................passed
Then I remove IP address "127.0.0.2" from loopback interface................passed
```

**Any additional information or context required to review the change:**

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by MikeDombo — COMMENTED** [review]
Blocked on https://github.com/aws-greengrass/aws-greengrass-testing/pull/220

File: uat/testing-features/src/main/resources/greengrass/features/ggmq-1.feature [hunk 1]
```diff
@@ -1463,6 +1463,115 @@ Feature: GGMQ-1
...
+    Then I add IP address "127.0.0.2" to loopback interface
+    Then I wait 90 seconds
+
+    And I discover core device broker as "default_broker" from "basic_connect" in OTF
+    And I connect device "basic_connect" on <agent> to "default_broker" using mqtt "<mqtt-v>"
+
+    And I disconnect device "basic_connect" with reason code 0
+    Then I remove IP address "127.0.0.2" from loopback interface
+
+    @mqtt3 @sdk-java
```
Comment (MikeDombo) [inline uat/testing-features/src/main/resources/greengrass/features/ggmq-1.feature:1534]: using only 1 client is perfectly fine for this test.

Comment (bgklika) [reply uat/testing-features/src/main/resources/greengrass/features/ggmq-1.feature:1534]: Which client's version v5 or v3 you prefer to keep ?

Comment (MikeDombo) [reply uat/testing-features/src/main/resources/greengrass/features/ggmq-1.feature:1534]: java v5.

Comment (auarbekov) [reply uat/testing-features/src/main/resources/greengrass/features/ggmq-1.feature]: fixed in 4a470d128157e6eccd9de7394fffe03fae8c3418

**Review by MikeDombo — APPROVED** [review]

**Pull request merged**